app/main.py

```diff
@@ -1,8 +1,28 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
Avoid using class variables for storing instances of the class. This can lead to unexpected behavior and makes the code harder to test. Consider using an instance variable or a separate data structure for this purpose.